vocode/streaming/telephony/conversation/outbound_call.py

```diff
@@ -140,7 +140,7 @@ def start(self):
             )
         else:
             raise ValueError("Unknown telephony client")
-        self.config_manager.save_config(self.conversation_id, call_config)
+        await self.config_manager.save_config(self.conversation_id, call_config)
```

Comment:
can this be `asyncio.create_task`? so that `start` can remain non-async and the outward API remain the same

or will we run into bugs like "no async context"